### PR TITLE
Mywatches refactor

### PR DESCRIPTION
**Pull request by kojoty**
This PR contains refactoring of my-watches-list and map refactoring.

It also introduce:
- list-of-caches chunk + basic set of columns - this could be a replacement for other lists  
- dynamic-map chunk + last-log variant - the first try of universal map widget

@deg-pl, @andrixnet, @harrieklomp this PR contains:
- translation changes
- sql-alter which should be applied after merge of this PR

### CODE REVIEW

File: lib/Objects/BaseObject.php [hunk 1]
```diff
@@ -28,4 +28,23 @@ public static function getCurrentUser(){
         return ApplicationContainer::Instance()->getLoggedUser();
     }
 
+    protected static function callOkapi($service, $params){
+
+        require_once( __DIR__.'/../../okapi/facade.php');
```
Comment (deg-pl) [inline lib/Objects/BaseObject.php]: Should be Facade, not facade (first capital letter)

**Comment by deg-pl** [comment]
Very good job! After fix "facade" -> "Facade" remove label don't commit.

**Comment by kojoty** [comment]
@andrixnet, @harrieklomp - could you review it (especially translations)? Can I commit these changes?

**Comment by harrieklomp** [comment]
I translate after commit on OCNL

**Comment by andrixnet** [comment]
Will handle this tonight or in the morning.

**Comment by andrixnet** [comment]
Feel free to merge. I will translate afterwards, just like harrieklomp

**Comment by andrixnet** [comment]
SQL changes applied at OCRO with the following remarks: 

`ALTER TABLE user DROP COLUMN cache_watches;` yielded error because of missing column.

**Comment by kojoty** [comment]
@deg-pl , @harrieklomp please don't forget to apply sqlalter form this PR

**Comment by harrieklomp** [comment]
sqlalter done on OCNL. No errors!

**Comment by deg-pl** [comment]
sqlalter done on OC PL and OC UK.

**Pull request merged**